src/main/kotlin/org/rust/lang/core/psi/impl/RustRawStringLiteralImpl.kt

```diff
@@ -4,6 +4,15 @@ import com.intellij.psi.tree.IElementType
 import org.rust.lang.core.psi.RustLiteralTokenType
 
 class RustRawStringLiteralImpl(type: IElementType, text: CharSequence) : RustTextLiteralImplBase(type, text) {
+    private var _hashes: Int = 0
+
+    val hashes: Int
+        get() {
+            @Suppress("UNUSED_VARIABLE")
+            val o = offsets // ensure #computeOffsets() has been called
```

Comment:
https://github.com/intellij-rust/intellij-rust/pull/621#r75926452, can add a test to https://github.com/intellij-rust/intellij-rust/blob/master/src/test/resources/org/rust/ide/annotator/fixtures/items/invalid_module_declarations.rs